ods_ci/tests/Resources/OCP.resource

```diff
@@ -200,3 +200,29 @@ Wait Until CRD Exists
     [Arguments]    ${crd_fullname}
     Wait Until Keyword Succeeds    15 times    5s
     ...    Oc Get    kind=CustomResourceDefinition    name=${crd_fullname}
+
+Resource Should Exist
+    [Documentation]    Check resource existson a namespace
+    [Arguments]       ${resource}     ${resource_name}    ${namespace}
+    ${rc}    ${out}=     Run and Return Rc And Output
+    ...  oc get ${resource} ${resource_name} -n ${namespace}
+    Should Be Equal    "${rc}"    "0"    msg=${out}
+
+Resource Status Should Be
+    [Documentation]    Check Resource Status
+    [Arguments]       ${oc_get}     ${resource}    ${expected_status}
+    ${status}=     Run
+    ...  ${oc_get}
+    Log    ${status}    console=True
+    Should Be Equal    ${status}    ${expected_status}   msg=${resource} is not in Ready status
+
+Check For Errors On Operator Logs
```

Comment:
Can you modify this keyword (in this PR or a follow-up on) so it can be used on any pod and not just the operator pod?

```
Verify Pod Logs Does Not Contain
[Arguments]    ${pod_name}  ${label_selector}  ${all_containers}=${FALSE}  ${container_name}=${EMPTY}    ${namespace}    ${since}  ${since-time}  ${regex_pattern}
```

And use the oc command instead of Oc Get. If you run oc logs -h you'll see that it has most of the parameters

Reply:
ok Jorge I modify it